graylog2-web-interface/src/components/nodes/NodesList.jsx

```diff
@@ -9,20 +9,20 @@ import { Spinner, EntityList, Pluralize } from 'components/common';
 import StoreProvider from 'injection/StoreProvider';
 import NodeListItem from './NodeListItem';
 
-const NodesStore = StoreProvider.getStore('Nodes');
 const ClusterOverviewStore = StoreProvider.getStore('ClusterOverview');
 
 const NodesList = createReactClass({
   displayName: 'NodesList',
 
   propTypes: {
     permissions: PropTypes.array.isRequired,
+    nodes: PropTypes.array.isRequired,
```

Comment:
Do we know what this array should contain? Perhaps we can use `arrayOf` instead?